src/components/GoodList.jsx

```diff
@@ -0,0 +1,9 @@
+import { GoodProduct } from './GoodProduct';
+
+export const GoodList = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <GoodProduct good={good} key={good.id} />
+    ))}
+  </ul>
+);
```

Comment:
There was nothing about creation of separate components and putting it into separate files in task description. But I ain't going to consider this as any kind of mistake. At the end of the day we are learning to use best practices, aren't we?)

Reply:
Yes, it wasn`t in the task about separate components. But I decided to use it in this task for extra practice))